src/Compnents/Person.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Person.scss';
+
+const Person = (props) => {
+  const partnerType = props.person.sex === 'm'
+    ? 'wife\'s'
+    : 'husband\'s';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          My name is
+          {props.person.name}
+        </h2>
+        <p className="Person__age">
+          {props.person.age ? `I am ${props.person.age}` : ''}
+        </p>
```

Comment:
You shouldn't show this paragraph in html if age is not defined
```suggestion
        {age && (
          <p className="Person__age">
            `I am ${props.person.age}`
          </p>
       )}
```